integrated-developer-portal/README.md

```diff
@@ -0,0 +1,132 @@
+3. Click + to add a new API product to the catalog
+4. Select the sample-integrated-developer-portal-product product and click next
+5. Configure as shown below
+- Published: Select published (checked)
+- Display title: Leave default name, sample-integrated-developer-portal-product
+- Display description: A portal for an API key protected proxy
+- Require developers to specify a callback URL: Keep deselected (unchecked)
+- Audience: Anonymous users (anyone can view)
+- API product image: Image of your choice
+- API documentation: Use the [integrated-developer-portal.yaml](integrated-developer-portal.yaml) OpenAPI document from this repo:
+    - Download [integrated-developer-portal.yaml](integrated-developer-portal.yaml) to your local computer
+    - Make note of your Apigee domain from the Apigee dashboard at Admin > Environments > Groups
+    - Replace "\[YOUR_DOMAIN\]" with your Apigee domain
+    - Upload your updated integrated-developer-portal.yaml as API documentation
+6. Scroll up and click save
+
+## Test Integrated Developer Portal
+
+Now that we have a developer portal, let's walk through it's workflow. First we'll create our create our developer account and sign in, then we'll create a Apigee app complete with a client id and secret, then we'll use the client id to authorize our requests and test our API. To do so, follow the steps below: 
+
+1. Navigate to your newly created portal. Portals > Sample Integrated Developer Portal
+2. Enter the portal by clicking the Live Portal button at the top right, or with the following URL: https://\[APIGEE-ORG\]-sampleintegrateddeveloperportal.apigee.io
+3. Click the Sign In button and create an account. This will necessitate an email confirmation. Note: this creates an Apigee developer account
+4. Navigate back to your portal's homepage (https://\[APIGEE-ORG\]-sampleintegrateddeveloperportal.apigee.io) and make sure that you're signed in
+5. Open the dropdown menu by clicking on your account and select Apps
+6. Select +NEW APP and update the following fields. Note: this creates an Apigee App for your developer
+- App Name: Sample App
+- APIs: Enable sample-integrated-developer-portal-product
+7. Click the SAVE button
+8. Click into APIs from the top navbar
+9. Click into your sample-integrated-developer-portal-product
+10. Select AUTHORIZE and select your app from the dropdown. This authorizes all of your portal requests with your App's API key
+11. Select the / GET request Path
+12. Under Try this API, click the EXECUTE button
+13. You should receive a 200 OK response from Apigee. The body of the response is a copy of the request data you sent to Apigee
+
+## Cleanup
+
+After you create your integrated developer portal you can clean up the artefacts from this sample in your Apigee Organization. First source your `env.sh` script, and then run
+
+```bash
+./clean-up-integrated-developer-portal.sh
+```
+
+After this, you need to manually delete manually created Apigee resources
```

Comment:
change to "After this, you need to manually delete the created Apigee resources"